src/components/Landing/index.tsx

```diff
@@ -169,29 +161,16 @@ export default function Landing() {
 
                 <div className="hide-mobile">
                   <div style={{ display: 'inline-block' }}>
-                    {v2Enabled ? (
-                      <Link
-                        to={'/create'}
-                        onClick={() => {
-                          window.fathom?.trackGoal('IIYVJKNC', 0)
-                        }}
-                      >
-                        <Button type="primary" size="large">
-                          <Trans>Design your project</Trans>
-                        </Button>
-                      </Link>
-                    ) : (
-                      <Button
-                        type="primary"
-                        size="large"
-                        onClick={() => {
-                          window.fathom?.trackGoal('IIYVJKNC', 0)
-                          scrollToCreate()
-                        }}
-                      >
+                    <Link
+                      to={'/create'}
+                      onClick={() => {
+                        window.fathom?.trackGoal('IIYVJKNC', 0)
```

Comment:
unrelated to this PR: should we turn the `IIYVJKNC` into a constant?

Reply:
Yeah def, this tracking stuff needs to be centralized into a lib/service somewhere. https://github.com/jbx-protocol/juice-interface/issues/986